docs/suspensive.org/src/pages/docs/jotai/Atom.en.mdx

```diff
@@ -0,0 +1,57 @@
+import { Callout } from '@/components'
+
+# Atom
+
+<Callout type='experimental'>
+
+`<Atom/>` is experimental feature, this interfaces could be changed
+
+</Callout>
+
+The Atom component provides the same props interface as Jotai's [useAtom](https://jotai.org/docs/core/use-atom#useatom) hook and can be used declaratively.
+
+### props.atom
+
+You can use Jotai's atom as is.
+
+```tsx /Atom/
+import { Atom } from '@suspensive/jotai'
+import { atom } from "jotai";
+
+const countAtom = atom(1);
+
+const Example = () => (
+  <Atom atom={countAtom}>
+    {([count, setCount]) => (
+      <>
+        <div>count: {count}</div>
+        <button onClick={() => setCount(count + 1)}>+1</button>
+      </>
+    )}
+  </Atom>
+)
+```
+
+For Async Atom, you can handle the loading state using Suspense.
+
+```tsx /Atom/
+import { Atom } from '@suspensive/jotai'
+import { Suspense } from '@suspensive/react'
+import { atom } from "jotai";
+
+const countAtom = atom(1)
+const asyncDoubleCountAtom = atom(async (get) => {
+  await delay(2000)
+  return get(countAtom) * 2
+})
+
+const Example = () => (
+    <Suspense fallback={'pending...'}>
+      <Atom atom={asyncDoubleCountAtom}>
+        {([count]) => (
+          <>count: {count}</>
+        )}
+      </Atom>
+    </Suspense>
+)
+```
```

Comment:
It seems that the motivation section is missing.

Reply:
> It seems that the motivation section is missing.

I was planning to add the Korean documentation once it's finished. I'll make sure to include it and not forget. Thank you.